src/Symfony/Component/Console/Command/Command.php

```diff
@@ -623,4 +636,60 @@ private function validateName($name)
             throw new InvalidArgumentException(sprintf('Command name "%s" is invalid.', $name));
         }
     }
+
+    /**
+     * Gets whether or not the auto lock option is enabled.
+     *
+     * @return bool
+     */
+    public function isAutoLock()
+    {
+        return $this->autoLock;
+    }
+
+    /**
+     * Sets the auto lock option.
+     *
+     * @param bool $autoLock
+     *
+     * @return Command The current instance
+     */
+    public function setAutoLock($autoLock)
+    {
+        $this->autoLock = (boolean) $autoLock;
```

Comment:
I think the standard is to use `(bool)`, though usually the values are not cast like that afaik